src/utils/setConstants.ts

```diff
@@ -0,0 +1,90 @@
+import path from 'path'
+import { getInstalledPath } from 'get-installed-path'
+import { getLocalOrGlobalConfig } from './config'
+import { getAbsolutePath } from './utils'
+
+export const setConstants = async () => {
```

Comment:
It'd be good if we can have tests for this.

Reply:
yea

Reply:
Added